### PR TITLE
Added more translatable strings in code. Updated template tools.pot.

### PR DESCRIPTION
**Pull request by DmitriySalnikov**
also improved Russian localization.

> akien-mga 
> Note that to avoid such hard to fix conflicts for a future PR, I'd advise you to do first a commit to change the source code, and only then make a second commit to regenerate the .pot file and update all .po files (using make all).

I do not understand how to update .pot and all .po files. Sorry.

### CODE REVIEW

File: tools/editor/project_settings.cpp [hunk 1]
```diff
@@ -81,10 +81,10 @@ void ProjectSettings::_notification(int p_what) {
 
 		translation_list->connect("button_pressed",this,"_translation_delete");
 		_update_actions();
-		popup_add->add_icon_item(get_icon("Keyboard","EditorIcons"),"Key",InputEvent::KEY);
-		popup_add->add_icon_item(get_icon("JoyButton","EditorIcons"),"Joy Button",InputEvent::JOYSTICK_BUTTON);
-		popup_add->add_icon_item(get_icon("JoyAxis","EditorIcons"),"Joy Axis",InputEvent::JOYSTICK_MOTION);
-		popup_add->add_icon_item(get_icon("Mouse","EditorIcons"),"Mouse Button",InputEvent::MOUSE_BUTTON);
+		popup_add->add_icon_item(get_icon("Keyboard","EditorIcons"),TTR("Key "),InputEvent::KEY);//"Key " - because the word 'key' has already been used as a key animation
```
Comment (akien-mga) [inline tools/editor/project_settings.cpp]: I understand what you're trying to achieve here, but I don't think it's the right way to do. We need a proper way to differentiate words that might need two different translations based on the context where they're used.

**Comment by akien-mga** [comment]
> I do not understand how to update .pot and all .po files. Sorry.

If you are on Linux or OSX with gettext installed, you can use the Makefile I put in ``tools/translations``:
```
cd tools/translations
make update   // regenerates the .pot file
make merge    // merges changes in all .po files, fuzzying the strings that changed, etc.
// "make" alone will do both "make update" and "make merge" at once
```

If you're not on Windows, it likely wouldn't work as easily. I'm not really familiar with using gettext on Windows but I guess the commands could be translated to run with the corresponding .exe.

**Comment by akien-mga** [comment]
See 6cd39ed for an example of how running ``make`` updated the tools.pot and PO files after your last PR.

**Comment by DimOkGamer** [comment]
@akien-mga finally, after several hours I was able to use 'make' with no errors)
cmd_2016-05-30_23-57-47 2

**Comment by akien-mga** [comment]
Awesome! (that you managed, not that it took several hours! :D)
Is there a way we could improve the Makefile to be more Windows-friendly?

**Comment by DimOkGamer** [comment]
I don't know. It's simple. You can only place somewhere the link to the necessary tools.

1. Download and install MinGw with msys extension: [link](https://sourceforge.net/projects/mingw/files/Installer/mingw-get-setup.exe/download)
2. Download and install [gettext](https://sourceforge.net/projects/gnuwin32/files/latest/download)
3. Add environment variables for the Bin folders
a) MinGW\msys\1.0\bin
b) Program Files (x86)\GnuWin32\bin
4. Make!

And besides, we have you)

**Pull request merged**